ast/dml.go

```diff
@@ -764,6 +764,8 @@ type SelectStmt struct {
 	IsAfterUnionDistinct bool
 	// IsInBraces indicates whether it's a stmt in brace.
 	IsInBraces bool
+	// QueryBlockOffset indicates the order of this SelectStmt if counted from left to right in the sql text.
+	QueryBlockOffset int
```

Comment:
It's set in TiDB?

Reply:
Yes, see https://github.com/pingcap/tidb/pull/11861/files#diff-59f10b93675b42905c972e190d2f059dR45